wiki/Beatmap_ranking_procedure/de.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - qualification
+  - nomination
+  - nominations
+  - nom
+  - ranking
+  - ranked
+  - Qualifizierung
+  - Nominierung
+  - gerankt
+---
+
+# Ranking-Verfahren für Beatmaps
+
+*Siehe auch: [Rank (Begriffsabgrenzung)](/wiki/Disambiguation/Rank)*
+
+Bevor [Beatmaps](/wiki/Beatmap) in die Kategorie [Ranked](/wiki/Beatmap/Category#ranked) eingestuft werden können, müssen sie von der Community überprüft werden, indem sie das Ranking-Verfahren durchlaufen.
+
+## Feedback
+
+Der Ersteller einer Beatmap kann seine Einsendungen als `Work in Progress` oder `Ausstehend` kennzeichnen. Die osu!-Community soll Feedback zu Beatmaps geben, die den beiden Kategorien zugeordnet sind.
+
+Durch [Modding](/wiki/Modding) erhält man konstruktive Kritik für eine Beatmap und kann so die Qualität der Beatmap steigern. Üblicherweise veröffentlichen Nutzer ihr Feedback auf der [Diskussionsseite](/wiki/Beatmap_discussion) einer Beatmap oder reden mit dem Ersteller direkt über Probleme.
+
+Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
```

Comment:
The direct translation of the original phrase

> Newer beatmappers usually need extensive modding to create beatmaps that meet the Ranked category's quality standards, and often they're required to recreate their beatmaps from scratch.

sounds a bit "despektierlich" to me, so I decided to go with this.

Reply:
I'm not against this change. But I think it still makes sense to add that inexperienced mappers might need to make a lot of changes to their submissions.

Reply:
```suggestion
Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Unerfahrene Beatmap-Ersteller müssen ihre Beatmaps potentiell an vielen Stellen überarbeiten. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
```

Alternative or as an addition to the paragraph:

```suggestion
Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Für die Erstellung qualitativ hochwertiger Beatmaps ist viel Erfahrung notwendig. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
```

Reply:
I like both

Reply:
Integrating both then.

```suggestion
Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Für die Erstellung qualitativ hochwertiger Beatmaps ist viel Erfahrung notwendig. Unerfahrene Beatmap-Ersteller müssen ihre Beatmaps potentiell an vielen Stellen überarbeiten. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
```